Add docs for daml script --ide-ledger (#208)
digital-asset/docs.daml.com: docs/2.7.0/docs/daml-script/index.rst

.. _daml-script-distributed:

Use Daml Script with the IDE Ledger
===================================

Similarly to running ``daml test`` or the codelens in VSCode, you can use

> ``daml script`` to run the scripts in a given dar file within the IDE Ledger,

Reviewer: ```suggestion
``daml script`` to run the scripts in a given DAR file within the IDE Ledger.
```

> a fully in-memory child process of ``daml script``, allowing you to quickly

Reviewer: ```suggestion
This is a fully in-memory child process of ``daml script``, allowing you to quickly
```

invoke a script without having to spin up a ledger in the background.

To run ``daml script`` in this mode, you should provide the ``--ide-ledger`` flag.
This flag is not compatible with ``--ledger-host``, ``--ledger-port``,

> ``--participant-config`` (as you'll see in the next section), and ``--json-api``.

Reviewer: ```suggestion
``--participant-config`` (described more in the next section), and ``--json-api``.
```

Note that since this uses an in-memory ledger, no state will be preserved once

> the script finished, you'll only receive a success flag and optionally the script result

Reviewer: ```suggestion
the script finishs. You will only receive a success flag and, optionally, the script result
```

> if using ``--output-file``.

Reviewer: ```suggestion
if you use ``--output-file``.
```
